code/stat159lambda/config.py

```diff
@@ -9,16 +9,16 @@
 
 USE_CACHED_DATA = os.environ.get('STAT159_CACHED_DATA', 'True') == 'True'
 
-NUM_PROCESSES = int(os.environ.get('STAT159_NUM_PROCESSES', 5))
 
 # fMRI data specific constants
-
 NUM_OFFSET_VOLUMES = 9
 
 NUM_VOLUMES = 3543
 
 NUM_VOXELS = 1108800
```

Comment:
If you wanted, you could calculate this constant using VOXEL_DIMENSIONS.  But it's fine as it is too